Dockerfile

```diff
@@ -1,14 +1,139 @@
+# sure our final image contains only what we've just built:
+RUN set -xe \
+	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/releases/download/OTP-${BUILD_VERSION}/otp_src_${BUILD_VERSION}.tar.gz" \
+	&& OTP_DOWNLOAD_SHA256="${BUILD_SHA256}" \
+	&& fetchDeps=' \
+		curl \
+		ca-certificates' \
+	&& apt-get update \
+	&& apt-get install -y --no-install-recommends $fetchDeps \
+	&& curl -fSL -o otp-src.tar.gz "$OTP_DOWNLOAD_URL" \
+	&& echo "$OTP_DOWNLOAD_SHA256  otp-src.tar.gz" | sha256sum -c - \
+	&& runtimeDeps=' \
+		libssl1.1 \
+	' \
+	&& buildDeps=' \
+		autoconf \
+		dpkg-dev \
+		gcc \
+		g++ \
+		make \
+		libncurses-dev \
+		libssl-dev \
+	' \
+	&& apt-get install -y --no-install-recommends $runtimeDeps \
+	&& apt-get install -y --no-install-recommends $buildDeps \
+	&& export ERL_TOP="/usr/src/otp_src_${OTP_VERSION%%@*}" \
+	&& mkdir -vp $ERL_TOP \
+	&& tar -xzf otp-src.tar.gz -C $ERL_TOP --strip-components=1 \
+	&& rm otp-src.tar.gz \
+	&& ( cd $ERL_TOP \
+	  && ./otp_build autoconf \
+	  && gnuArch="$(dpkg-architecture --query DEB_HOST_GNU_TYPE)" \
+	  && ./configure --build="$gnuArch" \
+	  && make -j$(nproc) \
+	  && make install ) \
+	&& find /usr/local -name examples | xargs rm -rf \
+	&& REBAR3_DOWNLOAD_URL="https://github.com/erlang/rebar3/archive/${REBAR3_VERSION}.tar.gz" \
+	&& REBAR3_DOWNLOAD_SHA256="a14711b09f6e1fc1b080b79d78c304afebcbb7fafed9d0972eb739f0ed89121b" \
+	&& mkdir -p /usr/src/rebar3-src \
+	&& curl -fSL -o rebar3-src.tar.gz "$REBAR3_DOWNLOAD_URL" \
+	&& echo "$REBAR3_DOWNLOAD_SHA256 rebar3-src.tar.gz" | sha256sum -c - \
+	&& tar -xzf rebar3-src.tar.gz -C /usr/src/rebar3-src --strip-components=1 \
+	&& rm rebar3-src.tar.gz \
+	&& cd /usr/src/rebar3-src \
+	&& HOME=$PWD ./bootstrap \
+	&& install -v ./rebar3 /usr/local/bin/ \
+	&& rm -rf /usr/src/rebar3-src \
+	&& apt-get purge -y --auto-remove $buildDeps $fetchDeps \
+	&& rm -rf $ERL_TOP /var/lib/apt/lists/*
+
+RUN set -xe \
+	    && apt update \
+	    && apt-get install -y --no-install-recommends \
+     	        libssl-dev make automake autoconf libncurses5-dev gcc \
+	        libdbus-1-dev libbz2-dev bison flex libgmp-dev liblz4-dev \
+	        libsodium-dev sed wget curl build-essential libtool git \
+		ca-certificates \
+	    && mkdir -p /opt/cmake \
+	    && wget -O /opt/cmake/cmake.tgz \
+	        https://github.com/Kitware/CMake/releases/download/v3.21.3/cmake-3.21.3-linux-x86_64.tar.gz \
+            && tar -zxf /opt/cmake/cmake.tgz -C /opt/cmake --strip-components=1
+
+# Install Rust toolchain
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+ENV CC=gcc CXX=g++ CFLAGS="-U__sun__" \
+    ERLANG_ROCKSDB_OPTS="-DWITH_BUNDLE_SNAPPY=ON -DWITH_BUNDLE_LZ4=ON" \
+    ERL_COMPILER_OPTIONS="[deterministic]" \
+    PATH="/root/.cargo/bin:/opt/cmake/bin:$PATH" \
+    RUSTFLAGS="-C target-feature=-crt-static"
+
+WORKDIR /usr/src/
+
+# Add our code
+RUN git clone https://github.com/syuan100/blockchain-node.git
```

Comment:
needs update. unless you're waiting on particular merges.

Reply:
Ah i see it below "fix-historic-balance". Hopefully will get @madninja to release his review block tonight or tomorrow AM.

https://github.com/helium/blockchain-node/pull/85

Reply:
Oops forgot to fix before pushing. Will add helium repo back after some final testing I'm doing.